core/build.gradle

```diff
@@ -86,6 +86,8 @@ dependencies {
 
     api group: "org.apache.calcite", name: "calcite-linq4j", version: calcite_linq4j_version
     api group: "org.apache.calcite.avatica", name: "avatica-core", version: avatica_core_version
+
+    compile group: 'nz.ac.waikato.cms.weka', name: 'weka-stable', version: '3.8.0'
```

Comment:
Please add a variable (`weka_version`) for the weka version in the `gradle.properties`.

Reply:
resolved with commit [89aff06](url)

Reply:
resolved in commit: [https://github.com/polypheny/Polypheny-DB/pull/211/commits/89aff06943a24226e43ad9d8007bd1415d395264](url)